### ISSUE TITLE
@Tag annotation not work as expected on the Controller class.

### ISSUE THREAD

**Issue by scruel** [springfox/springfox#3668]
It seems you decided to replace the old annotation @Api by @Tag, So after I read the docs of the @Tag class, I noticed that you described that "The annotation may be applied at class or method level, ..., for all operations of a class (when applied at class level)".
But when I tried to put this annotation on the Controller class like this:

```
@Tag(name = "test", description = "test")
public class IndexController 
```

It would not work as expected, the operations of the IndexController class would not apply the tag info which I applied on class.
I have to put @Tag annotation on all operations with same tag info one by one to achieve the goal of 'for all operations'. Otherwise, I will get two tag separately, which called 'index-controller' and 'test'. The 'test' tag will only contain the operations which putted on the same name @Tag annotation, the 'index-controller' tag will have the remains like this:
![image](https://user-images.githubusercontent.com/16933298/101281399-9cb68300-3809-11eb-9d36-5757f23f8d44.png)

Demo:
https://github.com/scruel/demo

**Comment by scruel** [comment]
"This is a springfox based project, not using spring-core for operation resolving, but its own mechanism. Please open a ticket there"

https://github.com/swagger-api/swagger-core/issues/3826

**Comment by AssimilatorEDF** [comment]
Seems like problem in next: in plugin, that set tags for operation from `@Api` annotation (`SwaggerOperationTagsReader`) there is a method:
![image](https://user-images.githubusercontent.com/13111701/101642800-fdda9280-3a65-11eb-8903-131a03dbe9fc.png)


It calls `context.findControllerAnnotation(Api.class);` that returns class annotation. But in plugin, that set tags for operation from `@Tag` annotation (`OpenApiOperationTagsReader`) there is method with same name, but it searches for `@Tags` / `@Tag` annotation only over a method neither a class
![image](https://user-images.githubusercontent.com/13111701/101642724-e6030e80-3a65-11eb-94e7-66f0ad8ca126.png)

**Comment by AssimilatorEDF** [comment]
Oh, already fixed in #3402 c:
